README.md

```diff
@@ -75,7 +75,14 @@ vars:
 ``` 
 
 ## (Optional) Step 5: Additional configurations
-<details><summary>Expand for configurations</summary>
+<details open><summary>Expand for configurations</summary>
+
+### Leveraging `orders` vs `orders` source
```

Comment:
Same suggestion that I think you meant for this to be the two different names
```suggestion
### Leveraging `orders` vs `order` source
```

Reply:
Thank you--updated!